.gitignore

```diff
@@ -7,6 +7,7 @@
 *.swo
 *.swp
 *.vi
+*.css
```

Comment:
Why/how did this work?

Reply:
I removed the reference with `git rm --cached <file>`, the gitignore change is just a re-organization choice